### DISCUSSION TITLE
Change background color in table

### DISCUSSION THREAD

**Discussion by andreabonetti** [marp-team/marp#532] [Q&A] [answered]
Hello,

I am working on a modified version of [rose-pine](https://github.com/rainbowflesh/Rose-Pine-For-Marp), you can find it [here](https://github.com/andreabonetti/md-slides/blob/main/marp/css/rose-pine-andrea.css).

I am wondering how I can change the background colors of table. So far, this code did not work:
```
table {
    background-color: var(--base);
}
```

What am I missing?

**Comment by yhatt** [comment, marked as answer]
The `default` theme (a base theme you are using) has different background colors in each row, so the specified background color into the table is overloaded by rows. You should set the background color into each cell to overload row colors.

```css
table td {
    background-color: var(--base);
}
```

**Reply by andreabonetti** [reply]
Thank you for your prompt reply! It works well :)
